test/e2e/kubectl/kubectl.go

```diff
@@ -2069,6 +2069,103 @@ metadata:
 			gomega.Expect(outBuiltIn).To(gomega.Equal(outStatusSubresource))
 		})
 	})
+
+	ginkgo.Describe("Kubectl rollout", func() {
+		deploymentYaml := `
```

Comment:
can you please add this to `testing-manifests/kubectl` and obtain it as in following example: https://github.com/kubernetes/kubernetes/blob/70a28449a76aaebe0f3707aa642499f47124dbae/test/e2e/kubectl/kubectl.go#L942